webapp/pubspec.yaml

```diff
@@ -8,6 +8,8 @@ dependencies:
   firebase: ^5.0.0
   http: ^0.12.0
   intl: ^0.16.0
+  dnd: ^1.4.0
+  uuid: ^2.0.0
   chartjs:
     git: https://github.com/marianamarasoiu/chartjs.dart.git
```

Comment:
This needs to be switched over to the Lark Systems hosted lib

Reply:
yes, good eye :) I've removed it for now since we don't use it, but when we do, we'll use that one